test/integration/aws/websocket.test.js

```diff
@@ -95,7 +95,7 @@ describe('AWS - API Gateway Websocket Integration Test', function () {
   });
 
   describe('Minimal Setup', () => {
-    it('should expose an accessible websocket endpoint', async function () {
+    it('should expose an accessible websocket endpoint', async () => {
```

Comment:
https://mochajs.org/#arrow-functions

Since arrow function is discouraged using with Mocha, should we consider converting all test functions a common standard?

I understand this does sound like a lot of work but it's something to consider.

Reply:
@sleepwithcoffee there's no point, as we generally do not use Mocha context. Our approach is to rely on arrow functions when we don't need to access context and otherwise stick to full functions

Reply:
@sleepwithcoffee I don't think we have an issue for that, and now I also hardly remember what was the issue. You may remove this variable handling temporarily and see if you get fails, that can at least hint us what's this guard for